Common/Extensions/NSBundle.swift

```diff
@@ -40,6 +40,10 @@ extension Bundle {
     var appStoreURL: String? {
         return object(forInfoDictionaryKey: "AppStoreURL") as? String
     }
+    
+    var isInvestigationalDevice: Bool {
+        return object(forInfoDictionaryKey: "IsInvestigationalDevice") as? String == "YES"
```

Comment:
Could this just look at the build flag, and avoid needing a plist to store it in?

Reply:
I just mirrored appName here. 

By build flag do you mean making it accessible via the scheme instead of through the xcconfig?

Reply:
MAIN_APP_DISPLAY_NAME needs to go in the plist for use by iOS. This doesn't need to go in the plist.  If you are already defining `IS_INVESTIGATIONAL_DEVICE` as a build config flag, then you can test for that directly in code, like we do feature flags.

Reply:
@ps2 Okay, I've updated this to use a feature flag!

Reply:
Thanks! Looks good!